lowcodelanguage/src/main/java/net/minecraftforge/fml/lowcodemod/LowCodeModContainer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.fml.lowcodemod;
+
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.IExtensionPoint;
+import net.minecraftforge.fml.ModContainer;
+import net.minecraftforge.fml.event.IModBusEvent;
+import net.minecraftforge.forgespi.language.IModInfo;
+import net.minecraftforge.forgespi.language.ModFileScanData;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Objects;
+
+import static net.minecraftforge.fml.loading.LogMarkers.LOADING;
+
+public class LowCodeModContainer extends ModContainer
+{
+    private static final Logger LOGGER = LogManager.getLogger();
+    private final ModFileScanData scanResults;
+    private Object modInstance;
+
+    public LowCodeModContainer(IModInfo info, ModFileScanData modFileScanResults, ModuleLayer gameLayer)
+    {
+        super(info);
+        LOGGER.debug(LOADING, "Creating LowCodeModContainer for {}", info.getModId());
+        this.scanResults = modFileScanResults;
+        this.modInstance = new Object();
+        this.contextExtension = () -> null;
+        registerExtensionPoint(IExtensionPoint.DisplayTest.class, () -> new IExtensionPoint.DisplayTest(()->this.modInfo.getVersion().toString(),
+                (incoming, isNetwork)-> true));
```

Comment:
This "noop" displaytest does not actually function for a lowcode mod present on the server but missing on the client. Since the client has no knowledge of the mod, it therefore has no knowledge of the `-> true` remoteVersionTest. So, the client displays it as missing and therefore adds the red X. This is the case that `NetworkConstants.IGNORESERVERONLY` is meant to solve, so it must be used here as it is special-cased on forge clients. And the `-> true` remoteVersionTest  should be kept for cases where a lowcode mod exists on the client but not on the server.